### PR TITLE
Add delete command for IPLookup module

### PR DESCRIPTION
**Pull request by levaitamas**
Mostly done as cargo cult programming, but it works for me.
Comments welcome.

### CODE REVIEW

File: core/modules/ip_lookup.cc [hunk 1]
```diff
@@ -302,6 +304,47 @@ CommandResponse IPLookup::CommandAdd(
...
+                          arg.prefix().c_str());
+  }
+
+  uint64_t prefix_len = arg.prefix_len();
+  if (prefix_len > 32) {
+    return CommandFailure(EINVAL, "Invalid prefix length: %" PRIu64,
+                          prefix_len);
+  }
+
+  net_mask = be32_t(~((1ull << (32 - prefix_len)) - 1));
```
Comment (melvinw) [inline core/modules/ip_lookup.cc]: nit: Could you use the [SetBitsLow()](https://github.com/NetSys/bess/blob/master/core/utils/bits.h#L193) utility here instead? There's an example of its usage with the big-endian types [here](https://github.com/NetSys/bess/blob/master/core/utils/ip.cc#L78)

**Comment by sangjinhan** [comment]
LGTM. Perhaps you can refactor the common code in `Add` and `Delete` regarding argument processing?

**Comment by levaitamas** [comment]
I tried to address  @melvinw's and @sangjinhan's  comments with a new commit. Thanks.

**Comment by sangjinhan** [comment]
Merged, thanks!
tumblr_m48dsbx5rt1qz4ueho1_1280

**Pull request merged**